src/dbt_score/cli.py

```diff
@@ -94,8 +94,20 @@ def cli() -> None:
     is_flag=False,
     default=None,
 )
+@click.option(
+    "--show",
+    help="Type of output which should be shown"
+    "when using `plain` as `--format`. "
+    "`all` shows all models and all rules"
+    "`failing-models` shows failing rules of failing models"
+    "`failing-rules` shows failing rules of all models"
+    "Default behavior is to show violated rules of all models.",
```

Comment:
```suggestion
    "Default is --failing-rules.",
```
This is clearer IMO, and avoids parsing 2 sentences with the same meaning :)